File: BRaVeMobile/app/src/main/java/intl/iom/bravemobile/models/activities/DistributionBinding.java
package intl.iom.bravemobile.models.activities;

import com.google.gson.annotations.SerializedName;

public class DistributionBinding {
    public int distributionId ;

    public int type ;

    //1- ONLINE, 2- OFFLINE, 3- HYBRID
    public int mode = 1;
    @SerializedName("biometricConfirmation")
    public boolean biometricReceiptRequired = true;
    @SerializedName("photoConfirmation")
    public boolean photoReceiptRequired = true;


}

File: BRaVe-Mobile-Backend/Models/Activity_Management/DistributionBinding.cs
namespace BRaVe_Mobile_Backend.Models
{
    public class DistributionBinding
    {
        public int DistributionId { get; set; }
        public int type { get; set; }
        public int mode { get; set; }

        public bool photoConfirmation { get; set; } = true;

        public bool biometricConfirmation { get; set; } = true;

        public DistributionBinding() { }

        /*public DistributionBinding(int distributionId, int type)
        {
            DistributionId = distributionId;
            this.type = type;
        }*/
    }
}